init.lua

```diff
@@ -459,13 +459,16 @@ local function render_ast(node)
     if node.no_prepend then
         res[#res + 1] = {type = "no_prepend"}
     end
-    if node.fbgcolor then
-        -- TODO: Fix this
+    if node.fbgcolor or node.bgcolor or node.bg_fullscreen then
         res[#res + 1] = {
             type = "bgcolor",
-            bgcolor = node.fbgcolor,
-            fullscreen = true
```

Comment:
This is a breaking change. fbgcolor ~= bgcolor

Reply:
minetest defaults to `bgcolor.fullscreen = true`. This is also breaking.

Reply:
(also should be noted that neither were documented features in flow, so it might not matter anyway)